src/bundle/Resources/views/dashboard/my_content_section.html.twig

```diff
@@ -0,0 +1,26 @@
+<section class='ez-tabs ez-dashboard-tabs'>
```

Comment:
it's almost the same template as all_content_section.html.twig, this leads me thinking that we should probably have a `TabBasedSection` class/interface somewhere